packages/notebook/src/actions.tsx

```diff
@@ -2435,12 +2437,17 @@ namespace Private {
     );
   }
 
+interface IInitDialog {
+  dialogShown: boolean;
+}
+
```

Comment:
Instead of introducing a mutable, what about looking if any cell is a code cell to trigger the toast in `runCells` at line 2356 instead of here.